docs/self-managed/concepts/multi-tenancy.md

```diff
@@ -0,0 +1,86 @@
+---
+id: multi-tenancy
+title: "Multi-tenancy"
+sidebar_label: "Multi-tenancy"
+description: "Multi-tenancy allows you to re-use your Camunda installation."
+---
+
+:::caution
+Multi-tenancy is disabled by default and can be enabled by the use of environment variables. This feature should be
+enabled in all required components, see:
+
+- [Identity feature flags](../../../self-managed/identity/deployment/configuration-variables/#feature-flags)
+- [Zeebe multi-tenancy](../../../self-managed/zeebe-deployment/configuration/gateway-config/#zeebegatewaymultitenancy)
+- [Operate multi-tenancy](../../../self-managed/operate-deployment/operate-configuration/)
+- [Tasklist multi-tenancy](../../../self-managed/tasklist-deployment/tasklist-configuration/#multi-tenancy)
+- [Optimize multi-tenancy]($optimize$/self-managed/optimize-deployment/configuration/multi-tenancy/)
+- [Connectors multi-tenancy](../../../self-managed/connectors-deployment/connectors-configuration/#multi-tenancy)
+  :::
+
```

Comment:
I remember there is a global flag for helm charts that can be used, can we refer to this? @aabouzaid ?

Reply:
It's `global.multitenancy.enabled`.
All details about the parameters are here: [Camunda Helm chart parameters](https://github.com/camunda/camunda-platform-helm/tree/main/charts/camunda-platform#parameters).

Reply:
I wonder if we should add it to the docs, we have a separate Helm section [here](https://docs.camunda.io/docs/next/self-managed/platform-deployment/helm-kubernetes/overview/), and we don't document the flags there.